test/buster.js

```diff
@@ -1,29 +1,19 @@
 var config = module.exports;
 
-config.browser = {
-    environment: "browser",
+config.node = {
+    environment: "node",
     rootPath: "../",
-    libs: [
-        "node_modules/lolex/lolex.js",
-        "node_modules/samsam/lib/samsam.js",
-        "node_modules/formatio/lib/formatio.js"
-    ],
     sources: [
         "lib/sinon.js",
-        "lib/sinon/util/core.js",
-        "lib/sinon/typeOf.js",
         "lib/**/*.js"
     ],
-    testHelpers: [
-        "test/test-helper.js"
-    ],
     tests: [
-        "test/**/*-test.js"
+        "test/**/*.js"
```

Comment:
If you look at the node config in master, it has this and I didn't change it. This "change" is just an artifact of how git compares.